pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -146,6 +149,7 @@ private static long getRandomInitialDelayInSeconds() {
     private static final int DEFAULT_STATUS_CONTROLLER_WAIT_FOR_PUSH_TIME_IN_SECONDS = 10 * 60; // 10 minutes
     private static final int DEFAULT_TASK_MANAGER_FREQUENCY_IN_SECONDS = -1; // Disabled
     private static final int DEFAULT_MINION_INSTANCES_CLEANUP_TASK_FREQUENCY_IN_SECONDS = 60 * 60; // 1 Hour.
+    private static final int DEFAULT_MINION_INSTANCES_CLEANUP_TASK_MAX_OFFLINE_TIME_RANGE_IN_SECONDS = 10 * 60; // 10 minutes
```

Comment:
Default should be 0 to preserve current behavior

Reply:
Think of it a bit, 0 would be too short, since this race condition could happen even for instances from k8s or docker.

Reply:
I guess I was thinking not to change the behavior now, assuming that whoever wanted to delete the znode wanted it done right away. I don't have a problem with whatever default. Just take the config in a human readable form please

Reply:
I think if we want to make this config human readable, we should do the same for all the other configs of periodic task as well, which are already using time in seconds format, since they are all under `ControllerPeriodicTasksConf` class. So if we want to clean this up, we should do it all together. For now using the same format should be sufficient.

Reply:
Created an issue to make all the configs of controller periodic tasks human readable:
https://github.com/apache/incubator-pinot/issues/7156